src/index.html

```diff
@@ -1,13 +1,421 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Title</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <title>Creative Bakery</title>
+    <link rel="icon" href="./images/cookie.svg">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;700&family=Poppins:wght@400;600;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles/main.scss">
   </head>
-  <body>
-    <h1>Hello Mate Academy</h1>
+  <body class="page__body">
+    <header class="page__header header">
+      <div class="container">
+        <div class="header__content">
+          <div class="header__top">
+            <a href="#" class="logo">
+              <img
+                 src="./images/logo-black.svg"
+                 alt="Bakerlab logo"
+                 class="logo__image"
+              >
```

Comment:
Should be only 2 spaces for attributes. Need to fix it everywhere
```suggestion
              <img
                src="./images/logo-black.svg"
                alt="Bakerlab logo"
                class="logo__image"
              >
```